test/conformance/kernel/urKernelGetGroupInfo.cpp

```diff
@@ -3,56 +3,156 @@
 // See LICENSE.TXT
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+#include "ur_api.h"
 #include <array>
 #include <uur/fixtures.h>
 
-using urKernelGetGroupInfoTest =
-    uur::urKernelTestWithParam<ur_kernel_group_info_t>;
-
-UUR_TEST_SUITE_P(
-    urKernelGetGroupInfoTest,
-    ::testing::Values(UR_KERNEL_GROUP_INFO_GLOBAL_WORK_SIZE,
-                      UR_KERNEL_GROUP_INFO_WORK_GROUP_SIZE,
-                      UR_KERNEL_GROUP_INFO_COMPILE_WORK_GROUP_SIZE,
-                      UR_KERNEL_GROUP_INFO_LOCAL_MEM_SIZE,
-                      UR_KERNEL_GROUP_INFO_PREFERRED_WORK_GROUP_SIZE_MULTIPLE,
-                      UR_KERNEL_GROUP_INFO_PRIVATE_MEM_SIZE,
-                      UR_KERNEL_GROUP_INFO_COMPILE_MAX_WORK_GROUP_SIZE,
-                      UR_KERNEL_GROUP_INFO_COMPILE_MAX_LINEAR_WORK_GROUP_SIZE),
-    uur::deviceTestWithParamPrinter<ur_kernel_group_info_t>);
-
-struct urKernelGetGroupInfoSingleTest : uur::urKernelTest {
+struct urKernelGetGroupInfoFixedWorkGroupSizeTest : uur::urKernelTest {
     void SetUp() override {
+        program_name = "fixed_wg_size";
         UUR_RETURN_ON_FATAL_FAILURE(urKernelTest::SetUp());
     }
+
+    // This value correlates to work_group_size<8, 4, 2> in fixed_wg_size.cpp.
+    // In SYCL, the right-most dimension varies the fastest in linearization.
+    // In UR, this is on the left, so we reverse the order of these values.
+    std::array<size_t, 3> work_group_size{2, 4, 8};
 };
-UUR_INSTANTIATE_DEVICE_TEST_SUITE_P(urKernelGetGroupInfoSingleTest);
+UUR_INSTANTIATE_DEVICE_TEST_SUITE_P(urKernelGetGroupInfoFixedWorkGroupSizeTest);
 
-struct urKernelGetGroupInfoWgSizeTest : uur::urKernelTest {
+struct urKernelGetGroupInfoMaxWorkGroupSizeTest : uur::urKernelTest {
     void SetUp() override {
-        program_name = "fixed_wg_size";
+        program_name = "max_wg_size";
         UUR_RETURN_ON_FATAL_FAILURE(urKernelTest::SetUp());
     }
 
-    // This must match the size in fixed_wg_size.cpp
-    std::array<size_t, 3> wg_size{4, 4, 4};
+    // These values correlate to max_work_group_size<6, 5, 4> and
+    // max_linear_work_group_size<120> in max_wg_size.cpp.
```

Comment:
looks like the values in the comment got out of sync with the ones below

Reply:
Good catch, updated.